espmonitor/src/types.rs

```diff
@@ -73,6 +74,8 @@ impl Chip {
             Ok(Chip::ESP32S2)
         } else if target.contains("-esp8266-") {
             Ok(Chip::ESP8266)
+        } else if target.contains("-esp32c3-") {
+            Ok(Chip::ESP32C3)
```

Comment:
Isn't `target` here unfortunately just `riscv32imc-unknown-none-elf`, so we can't really match on this here?

Reply:
I removed this match, hopefully it is enough for now to indicate that chip can't be resolved by target in case of C3

Reply:
Yeah, I'll probably take a quick pass to improve the error message before I do the next release.